dhctl/cmd/dhctl/commands/bootstrap/phase.go

```diff
@@ -36,13 +38,13 @@ func DefineBootstrapInstallDeckhouseCommand(parent *kingpin.CmdClause) *kingpin.
 	app.DefineDeckhouseInstallFlags(cmd)
 
 	cmd.Action(func(c *kingpin.ParseContext) error {
-		var sshClient *ssh.Client
+		var nodeInterface node.Interface = local.NewNodeInterface()
 		if len(app.SSHHosts) != 0 {
```

Comment:
please keep only non local bootstrap for phases

Reply:
done